packages/data-broker/src/__tests__/DataBroker/DataBroker.test.js

```diff
@@ -73,14 +73,6 @@ describe('DataBroker', () => {
 
   describe('pull()', () => {
     describe('input validation', () => {
-      beforeAll(() => {
-        jest.spyOn(global.console, 'warn').mockImplementation();
-      });
-
-      afterAll(() => {
-        global.console.warn.restore();
-      });
-
```

Comment:
Ah good question! I removed it since I was getting an error about there being no `.restore()` on `global.warn`. I think this is all related to the jest upgrade. Truthfully I should've tracked down exactly why this didn't work anymore (I tried for a bit and then gave up), and how to support it in the new version, but I felt that maybe this wasn't the most important thing? I think it just suppresses a single warning that originates from a test where we do an unhappy path check...

Reply:
Sure thing - happy to leave this as is. Thanks for the explanation!